.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 orbs:
   cypress: cypress-io/cypress@1
 jobs:
-  build-and-test:
+  dependencies:
```

Comment:
what does this change do?

Reply:
nothing, it's a legacy of my moving stuff into workflows. Will fix :smile: